pipeline/notifications/check_message.py

```diff
@@ -0,0 +1,25 @@
+from typing import List
+
+import prefect
+from prefect import task
+
+from pipeline.types import Notification, User
+
+
+class Notifier:
+    def __init__(self):
+        pass
+
+    def check_msg_notifier(self, notif: Notification) -> List[User]:
+
+        # check if all users in match have a message
+        # if there are users who don't have a message, return a list of them for notification
+        users_with_msg = []
```

Comment:
Use a set here, since there is no need to store a user multiple times.

Otherwise, if there are a thousand messages in the chat, you might end up with a thousand values in `users_with_msg`.

Important to think about when to use list vs set vs dict vs tuple.